tools/run_tests/performance/scenario_config.py

```diff
@@ -446,6 +446,20 @@ def scenarios(self):
                 server_threads_per_cq=2,
                 categories=inproc_categories + [SCALABLE])
 
+            yield _ping_pong_scenario(
+                'cpp_protobuf_async_unary_qps_unconstrained_multiple_channels_%s'
+                % secstr,
+                rpc_type='UNARY',
+                client_type='ASYNC_CLIENT',
+                server_type='ASYNC_SERVER',
+                unconstrained_client='async',
+                secure=secure,
+                channels=8,
```

Comment:
btw, the "unconstrained" tests seem to be using 64 channels by default:
https://github.com/grpc/grpc/blob/be7df211f99a6cd3497bfd55203cb9e6801545f7/tools/run_tests/performance/scenario_config.py#L45, so I'm not sure this setting is useful. (or maybe we're actually using too many channel?)

Reply:
If `channels` are not specified, it is 64 by default. https://github.com/grpc/grpc/blob/be7df211f99a6cd3497bfd55203cb9e6801545f7/tools/run_tests/performance/scenario_config.py#L187

However, each channel map to a completion queue, and a completion queue maybe bound to one-or-more threads.